kube/int/destroy.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -x
-DAX_NAMESPACE=${DAX_NAMESPACE:-'dax-int'}
+DAX_NAMESPACE=${DAX_NAMESPACE:-'lsst-lsp-int-dax'}
 
 kubectl delete ingress dax-imgserv-ingress --namespace $DAX_NAMESPACE
 kubectl delete service dax-imgserv-service --namespace $DAX_NAMESPACE
```

Comment:
You're still not following the pattern with destroy_volumes.  Move these lines there

Reply:
How about now?

Reply:
Well this is better, but you didn't fix the names of the volume, so your pvc doesn't point to the right volume, you missed the rename there (and in the names of the yaml files)